build_and_test.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+#
+# Copyright 2017-present The kokoro iOS runner Authors. All Rights Reserved.
```

Comment:
Capitalization on the authors seems wrong.

Reply:
Done.